unify-front-end/components/profile/Profile.tsx

```
@@ -0,0 +1,269 @@
+      default:
+        return null;
+    }
+  };
+
+  const renderTabContent = useMemo(() => {
+    switch (activeTab) {
+      case 'Comments':
+        return (
+          <FeedWithHook
+            key={`comments-${userId}`}
+            useFeedHook={() => useCommentedOnFeed(userId)}
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)